front-end/src/services/scheduledLessonService.js

```diff
@@ -0,0 +1,23 @@
+import API from 'API';
+
+export const getAllScheduledLessons = async (id, minDate, maxDate) => {
```

Comment:
Pass params as object not as variables, because it always could increase.
See method below